idaes/generic_models/properties/examples/methanol_water_ideal.py

```diff
@@ -0,0 +1,221 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Phase equilibrium package for methanol synthesis using ideal VLE.
+Author: Brandon Paul
+"""
+# Import Python libraries
```

Comment:
There needs to be some sort of test for this property package if it is going to go in core. It should solve the model for at least one test case and check the values against some source (preferably literature).